api.md

```diff
@@ -287,6 +287,12 @@ in the table.
 to how you handle constraints. A hypertable can contain foreign keys to normal SQL table
 columns, but the reverse is not allowed. UNIQUE and PRIMARY constraints must include the
 partitioning key.
+
+>:WARNING: Running `create_hypertable` with data migration on a table with foreign key contraints to other tables 
```

Comment:
Maybe this should be a `NOTE` and not `WARNING` (if such a distinction exists).

Reply:
I don't have strong opinion and troubled to decide, which one to use. It would be nice to hear reasoning why one over another.

Reply:
I strongly second using "Note" instead of warning. These are potential issues that we want developers to be aware of rather than problems with the db. It would  be appropriate for an FAQ  section

Reply:
@gayyappan @erimatnor There is not such thing as note, only either `TIP` or `WARNING`. I can switch to tip.

Reply:
I guess tip makes sense, but don't feel strongly. Warning seems too alarming for this thing, IMO.